themes/V3/5ePHB/snippets/coverpage.gen.js

```diff
@@ -1,124 +1,123 @@
+  "In a forgotten land, a reporter and a spy try to stop the apocalypse.",
+  "In a forbidden land of prophecy, a scientist and an archivist oppose a cabal of barbarians intent on stealing the souls of the innocent.",
+  "On an infernal world of illusion, a grave robber and a watchman try to find revenge and combat a syndicate of mages intent on stealing the source of all magic.",
+  "In a galaxy of dark magic, four fighters seek freedom.",
+  "In an empire of deception, six tomb-robbers quest for the ultimate weapon and combat an army of raiders.",
+  "In a kingdom of corruption and lost souls, in an age of panic, eight planetologists oppose evil.",
+  "In a galaxy of misery and hopelessness, in a time of agony and pain, five planetologists search for vengance.",
+  "In a universe of technology and insanity, in a time of sorcery, a computer techician quests for hope.",
+  "On a planet of dark magic and barbarism, in an age of horror and blasphemy, seven librarians search for fame.",
+  "In an empire of dark magic, in a time of blood and illusions, four monks try to find the ultimate weapon and combat terrorism.",
+  "In a forgotten empire of dark magic, six kings try to prevent the destruction of mankind.",
+  "In a galaxy of dark magic and horror, in an age of hopelessness, four marines and an outlaw combat evil.",
+  "In a mysterious city of illusion, in an age of computerization, a witch-hunter tries to find the ultimate weapon and opposes an evil corporation.",
+  "In a damned kingdom of technology, a virtual reality programmer and a fighter seek fame.",
+  "In a hellish kingdom, in an age of blasphemy and blasphemy, an astrologer searches for fame.",
+  "In a damned world of devils, an alien and a ranger quest for love and oppose a syndicate of demons.",
+  "In a cursed galaxy, in a time of pain, seven librarians hope to avert the apocalypse.",
+  "In a crime-infested galaxy, in an era of hopelessness and panic, three champions and a grave robber try to solve the ultimate crime.",
 ];
 
+module.exports = () => {
+  return `
 
-module.exports = ()=>{
-	return `<style>
-  .page#p1{ text-align:center; counter-increment: none; }
-  .page#p1:after{ display:none; }
-  .page:nth-child(2n) .pageNumber { left: inherit !important; right: 2px !important; }
-  .page:nth-child(2n+1) .pageNumber { right: inherit !important; left: 2px !important; }
-  .page:nth-child(2n)::after { transform: scaleX(1); }
-  .page:nth-child(2n+1)::after { transform: scaleX(-1); }
-  .page:nth-child(2n) .footnote { left: inherit; text-align: right; }
-  .page:nth-child(2n+1) .footnote { left: 80px; text-align: left; }
-</style>
+{{coverPage
 
-{{margin-top:225px}}
+![image](https://i.imgur.com/Mqx8Vf7.png) {left:0px}
 
 # ${_.sample(titles)}
 
-{{margin-top:25px}}
+## ${_.sample(subtitles)}
+___ 
+
+{{banner
+5e HOMEBREW
+}}
+
+{{footnote
+Everything a Homebrew needs to create a pretty document <br> for the world's greatest roleplaying game
+}}
```

Comment:
```suggestion
{{footnote
Everything a Homebrew needs to create a pretty document\
for the world's greatest roleplaying game
}}
```

Reply:
i'm making changes to this text, and will just let it break naturally.